MMM-Volume.js

```diff
@@ -55,15 +55,15 @@ Module.register("MMM-Volume", {
         getVolumeScript: `amixer sget 'Digital' | grep -E -o '[[:digit:]]+%' | head -n 1| sed 's/%//g'`, // get 0~100
         setVolumeScript: `amixer sset -M 'Digital' #VOLUME#%`, // set 0~100
       },
- 	    "PULSE": {
+ 	"PULSE": {
         getVolumeScript: `amixer get Master  | awk -F"[][]" '{print ""$2""}' | grep %  | awk 'NR==1{print $1}' | awk '{gsub(/%/,"") ; print}'`, // get 0~100
         setVolumeScript: `amixer set Master #VOLUME#% -q`, // set 0~100
       },
-      "SPEAKER": {
+      "ReSpeaker_Speaker": {
         getVolumeScript: `amixer sget Speaker`,
```

Comment:
```
getVolumeScript: `amixer sget Speaker`,
```
This will return a bunch of text. getVolumeScript should return only integer value 0~100. That's why other prescript use `awk` or `grep` like, 
```
getVolumeScript: `amixer sget 'Digital' | grep -E -o '[[:digit:]]+%' | head -n 1| sed 's/%//g'`, // get 0~100
```

Reply:
I think just same command with other script be enough for your getVolumeScript (but not tested, because I have no  ReSpeaker)

Reply:
OK. I got it. 
Let me test for it.

Reply:
Yes, I test the command "amixer sget" with grep and sed parameter. The output information is as following:

```
pi@Raspberry:~ $ amixer sget Speaker
Simple mixer control 'Speaker',0
  Capabilities: pvolume
  Playback channels: Front Left - Front Right
  Limits: Playback 0 - 127
  Mono:
  Front Left: Playback 110 [87%] [-11.00dB]
  Front Right: Playback 110 [87%] [-11.00dB]
pi@Raspberry:~ $
pi@Raspberry:~ $
pi@Raspberry:~ $ amixer sget 'Speaker' | grep -E -o '[[:digit:]]+%' | head -n 1| sed 's/%//g'
87
pi@Raspberry:~ $
```

Reply:
So I modified the following scripts in MMM-Volume.js:
```
    presetScript: {
      ......
      "Respeaker_Speaker": {
        getVolumeScript: `amixer sget Speaker | grep -E -o '[[:digit:]]+%' | head -n 1| sed 's/%//g'`, // get 0~100
        setVolumeScript: `amixer sset Speaker #VOLUME#%`, // set 0~100
      },
      "Respeaker_Playback": {
        getVolumeScript: `amixer sget Playback | grep -E -o '[[:digit:]]+%' | head -n 1| sed 's/%//g'`, // get 0~100
        setVolumeScript: `amixer sset Playback #VOLUME#%`, // set 0~100
      }, 

```
During MagicMirror startup process, at first I saw a blue rectangle and later I saw the volume percentage. Are there any other test methods?

Reply:
Just execute your getVolumeScript on terminal shell (remove quotation marks of start and end), And let's see what is showing. If just 0-100 integer values be shown, it will be ok.

Reply:
The following output is OK?
```
pi@Raspberry:~ $ amixer sget 'Speaker' | grep -E -o '[[:digit:]]+%' | head -n 1| sed 's/%//g'
87
pi@Raspberry:~ $
```

Reply:
I think it's enough